demos/02 Supervised Classification/02b Training a Supervised Classifier.ipynb

```diff
@@ -726,10 +505,30 @@
     "Next move on to [02c Deploying a Supervised Classifier.ipynb](02c%20Deploying%20a%20Supervised%20Classifier.ipynb) to scale the inference of the model we just trained!"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "2d6ddfdc-df75-434f-8227-8a3db1281c53",
+   "metadata": {},
+   "source": [
+    "## Cleaning Up\n",
+    "It is always best practie to clean up after ourselves, including both Function results and unused Products:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6d4edefa-2758-4b4a-9f53-f57fbb98ede5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "async_func.delete_jobs(delete_results=True)\n",
```

Comment:
I don't think we should delete these here, since we try and use them in notebook 2c, right?

Reply:
this function is designed just to generate the training data and can be thrown away, the next notebook makes the function that deploys the model trained in this step